### PR TITLE
Issue #3178 - Update Basic Usage guide

### LINKED ISSUE #3178: Documentation - Basic usage - content update
I found some issues with the text in Basic Usage guide - https://docs.okd.io/latest/minishift/using/basic-usage.html

* Not needed bracket
  https://docs.okd.io/latest/minishift/using/basic-usage.html#persistent-configuration
> {The global configuration file is maintained at $MINISHIFT_HOME/config/global.json.
* Misleading information. If I use start flags, they are not set to persistent configuration
> By Default minishift persist start flags to peristent configuration, to disable it, use `minishift config set save-start-flags false`.
* Duplicate information about priorities of runtime options
  https://docs.okd.io/latest/minishift/using/basic-usage.html#unsetting-persistent-configuration-values

### PR DESCRIPTION
**Pull request by tsedmik**
Signed-off-by: Tomáš Sedmík <tsedmik@redhat.com>

Fixes #3178

### CODE REVIEW

**Comment by praveenkumar** [comment]
test this please.

File: docs/source/using/basic-usage.adoc [hunk 1]
```diff
@@ -188,16 +183,6 @@ For example:
...
-[NOTE]
-====
-The precedence for user-defined values is as follows:
-
-. Command line flags.
-. Environment variable.
-. Instance-specific configuration.
-. Global configuration.
-====
-
```
Comment (LalatenduMohanty) [inline docs/source/using/basic-usage.adoc]: I could not find where it is mentioned again. Can you please point me?

**Comment by agajdosi** [comment]
ok to test

**Comment by agajdosi** [comment]
retest this please

**Pull request merged**